.github/workflows/ci.yml

```diff
@@ -27,6 +31,27 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Git config
```

Comment:
If you install git client by apt at line 20 probably this step is unnecessary.

Reply:
Checkout action uses git client only if already installed, otherwise uses GitHub API to download files only.

Reply:
Hmmm
Don't know if I got it... I am installing git on line 20